src/server/routes/questions.ts

```diff
@@ -20,12 +21,16 @@ router.get('/questions', (req: Request<unknown, unknown, unknown, GetQuestionsBo
     res.status(404);
     return res.send({ errorMessage: 'questionCounts out of range(0 - 50).' });
   }
+
   const questions: Question[] = [];
-  for (let i = 0; i < questionCounts; i += 1) {
-    questions.push(question);
+  if (process.env.USING_MOCK === 'TRUE') {
```

Comment:
Can we remove 'TRUE' and use if (JSON.parse(process.env.USING_MOCK))?

Reply:
Sure, i will fix it